src/sentry/static/sentry/app/actionCreators/group.jsx

```diff
@@ -148,3 +148,19 @@ export function updateNote(api, group, note, id, oldText) {
 
   return promise;
 }
+
+export function deleteEventAttachment(api, url, groupId) {
+  const promise = api.requestPromise(url, {
+    method: 'DELETE',
+  });
+
+  promise.then(() => {
+    const currentGroup = GroupStore.get(groupId);
+
+    if (currentGroup && currentGroup.attachmentCount > 0) {
+      GroupStore.updateEventAttachmentsCount(currentGroup.attachmentCount - 1, groupId);
```

Comment:
Thinking about it again, this would probably be too slow. We might even have to remove the count alltogether, so let's leave it as is 👍

Reply:
Yeah, I've been thinking the same thing... If you delete an attachment on the event detail screen, this number will lower by one (in optimistic UI kind of way - because it might be expensive to refetch), but if you visit the new Attachments tab (or delete attachment there), this number will get updated with the truth from the server, because we are doing the fetch anyway.

Ok, I am leaving it for time being and we will see 👍